### ISSUE TITLE
Slow browse page load with multi-column foreign key

### ISSUE THREAD

**Issue by M393** [phpmyadmin/phpmyadmin#18728]
### Describe the bug

When browsing a table with a multi-column foreign key phpmyadmin adds links to the values in the foreign key columns.
These links only point to that columns part of the foreign key instead of using all parts.

phpmyadmin displays a tooltip for each link, and to get the data it makes one request per column and row in the parent table.
This negatively impacts performance on big tables where no index is available for the 2nd, 3rd, ... columns in the foreign key.

### To Reproduce

1. create a table with multi-column foreign key where the parent table has a lot of data
2. browse the table
3. It takes unusually long to display the data

### Expected behavior

The table should be displayed without much delay when the original query is using an index.
Ideally the links should point to the exact row (using all the key columns, instead of just the clicked column)

### Screenshots

![image](https://github.com/phpmyadmin/phpmyadmin/assets/56256405/f946de89-185c-425d-9c7f-b9ea78503287)

### Server configuration

- Operating system: linux
- Web server: apache2
- Database version: MySQL 5.7.42
- PHP version: 8.1.2
- phpMyAdmin version: 5.2.1

### Additional context

In this case the parent table has 2M rows, the page needs half a minute to load, if the number of rows displayed is increased the time needed increases proportionally.
It could also make sense to cache the looked up rows, because often the foreign keys are exactly the same. 
(See the screenshot 2nd and 3rd column)

**Comment by williamdes** [comment]
I agree with you, I tried to optimize this but did not succeed for now
It's also an issue for me on remote databases
We fetch the values one by one, not very great

**Comment by williamdes** [comment]
I am not sure but maybe it is this function: https://github.com/phpmyadmin/phpmyadmin/blob/QA_5_2/libraries/classes/InsertEdit.php#L1024

You can try to make it return before fetching the data to see if it speeds things up

**Comment by M393** [comment]
This is the function loading the data. Added `return null;` at the top and it loads fast.
https://github.com/phpmyadmin/phpmyadmin/blob/d59645d8fbabf3ac6b145840cde55d878114c07f/libraries/classes/Display/Results.php#L4337-L4370

**Comment by williamdes** [comment]
Thank you !
This confirms what I was thinking, we need to optimize this function to fetch all values at once if possible

**Comment by williamdes** [comment]
> I also stumbled across this problem. I assume fixing this is not trivial, but for me it would be fine if there were a $cfg option to disable this feature. As for now, I added the "return null" fix in Results.php.

https://github.com/phpmyadmin/phpmyadmin/issues/19045
